### PR TITLE
release email styling

### PR DESCRIPTION
**Pull request by DanielW1987**
So that's the status for now.
As I've learned, styling emails is a pretty annoying topic and there are a number of best practices for looking good across a variety of email clients. 

I have also already enabled scheduling.

### CODE REVIEW

**Comment by DanielW1987** [comment]
[Notification e-mail styling + release images](https://trello.com/c/90LHShQr/308-notification-e-mail-styling-release-images)

**Comment by DanielW1987** [comment]
[Test Notifications](https://trello.com/c/HmyP24sa/309-test-notifications)

File: webapp/src/main/resources/templates/email/fragments/releases-layout.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        </div>
+        <div id="content">
+            <h1>Hi <span th:text="${username}">username</span></h1>
+            <div layout:fragment="content"></div>
+            <div id="footer" class="text-muted text-center text-s">
+                <div id="unsubscribe">
+                    <p>
+                        This email was sent to you because you are registered as a member on metal-detector.rocks. Please do not reply
+                        to this email, as we cannot reply to you via this address. If you want to unsubscribe please go to the
+                        <a href="#" th:href="@{${notificationSettingsUrl}}">notification settings</a> of your account.
```
Comment (hedgehog1833) [inline webapp/src/main/resources/templates/email/fragments/releases-layout.html:82]:
```suggestion
                        to this email, as we cannot reply to you via this address. If you want to unsubscribe please go to your account's <a href="#" th:href="@{${notificationSettingsUrl}}">notification settings</a>.
```

Comment (hedgehog1833) [reply webapp/src/main/resources/templates/email/fragments/releases-layout.html:82]: Better also check the formatting here if you use the suggestion.

**Review by hedgehog1833 — APPROVED** [review]

**Pull request merged**